backend/inspirehep/records/utils.py

```diff
@@ -152,3 +154,16 @@ def remove_author_bai_from_id_list(author):
     author["ids"] = [
         author_id for author_id in author["ids"] if author_id["schema"] != "INSPIRE BAI"
     ]
+
+
+def is_document_scanned(file_data):
+    try:
+        with pdfplumber.open(file_data) as pdf:
+            for page in pdf.pages:
+                page_text = page.extract_text()
+                if page_text:
+                    return False
+    # case when the pdf is invalid
+    except PDFException:
+        pass
```

Comment:
let's add a log to have more info for the missing pdfs

Reply:
do we want to raise an error then (eg. `UnsupportedFileError`)? And what value of the`is_scanned` do we put in that case?